### DISCUSSION TITLE
Open Source Documentation Offline / "Performing Maintenance"

### DISCUSSION THREAD

**Discussion by leoedin** [syslog-ng/syslog-ng#4633] [General]
The syslog-ng Open Source documentation has been offline for more than a day now. It should be here: https://www.syslog-ng.com/technical-documents/list/syslog-ng-open-source-edition/

Is there another source for this? Does anyone know how long it's likely to be offline for?

**Comment by MrAnno** [comment]
The AxoSyslog documentation can be a good alternative:

https://axoflow.com/docs/axosyslog-core/

**Comment by czanik** [comment]
There is an open ticket for it inside One Identity, and it is being escalated right now. Unfortunately I still do not know when this will be fixed.

**Comment by czanik** [comment]
Search does not work yet, but documentation is available again on the syslog-ng.com website.
